app/src/main/java/ch/epfl/sdp/game/data/Prey.kt

```diff
@@ -0,0 +1,8 @@
+package ch.epfl.sdp.game.data
+
+class Prey(id: Int, var NFCTag: String = "") : Player(id) {
+    enum class PreyState{
```

Comment:
I'd extract this enum on a file of it's own as other classes may need it

Reply:
It is public so other classes may access it through Prey.PreyState as I did in the PredatorActivity. But it is heavy to read and I extracted it now.